test/common/parallel_tests.js

```diff
@@ -79,8 +79,8 @@ function parallelTests(tag, suite, section) {
 	}
 
 	// Looking recursevely for javascript files not containing the word "common"
-	var pathfiles = find.fileSync(/^((?!common)[\s\S])*.js$/, suiteFolder);
-	var initPathfiles = pathfiles.splice(0, maxParallelism);
+	var filepaths = find.fileSync(/^((?!common)[\s\S])*.js$/, suiteFolder);
```

Comment:
`filePaths`

Reply:
This is arguable https://english.stackexchange.com/questions/428215/filepath-or-file-path. To me it sounds ok as it is.

Reply:
Please check last comment in link that you provided. `Filepath is not a word in the English language.`

Reply:
I've checked with English speakers and it is fine for them as it is.

Reply:
Not sure if `\s` is required here due to specific library. As only `\S` can match the files.